fighthealthinsurance/ml/ml_router.py

```
@@ -68,27 +68,41 @@ def entity_extract_backends(self, use_external) -> list[RemoteModelLike]:
+            )
         else:
-            models = self.all_models_by_cost
+            models = self.internal_models_by_cost
         abstract_optional = ""
         text_optional = ""
         if abstract is not None:
-            abstract_optional = f"--- Current abstract: {abstract} ---"
+            abstract_optional = f"--- Current abstract: {abstract[0:1000]} ---"
         if text is not None:
-            text_optional = f"--- Full-ish article text: {text} ---"
+            text_optional = f"--- Full-ish article text: {text[0:1000]} ---"
```

Ruff (0.8.2)

Local variable `text_optional` is assigned to but never used

Remove assignment to unused variable `text_optional`

(F841)